examples/backtrace_while_sampling.rs

```diff
@@ -22,8 +22,9 @@ fn main() {
     }
 
     if let Ok(report) = guard.report().build() {
+        #[allow(unused_variables)]
         let file = File::create("flamegraph.svg").unwrap();
-        report.flamegraph(file).unwrap();
+        //report.flamegraph(file).unwrap();
```

Comment:
Why is this line commented? If it's not commented out, it seems that the `allow(unused_variables)` is not needed.